core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -247,14 +246,10 @@ private void parseArguments(final String[] args) {
             .describedAs("file name");
         forceOption = optionParser.accepts("force", "Force the removal of members of the consumer group (intended to remove stopped members if a long session timeout was used). " +
                 "Make sure to shut down all stream applications when this option is specified to avoid unexpected rebalances.");
-        executeOption = optionParser.accepts("execute", "Execute the command.");
```

Comment:
In contrast to the `--zookeeper` flag, there is no indicator that this option was deprecated... Do we feel confident to remove it? Seems we missed to update this one when implementing KIP-171.

Reply:
Hmm...awkward. Yeah I think we should probably not remove it yet in that case. I'll add the deprecation note in this PR, and then we can remove it in 4.0 (is 4.0 coming after 3.0 or will we have 3.1?)

Reply:
@mjsax PR is ready for review again, lmk what you think

Reply:
I think we would have a 3.1 and afterwards 4.0.